libs/blocks/modal/modal.css

```diff
@@ -1,3 +1,8 @@
+:root {
+  /* Modal Colors */
```

Comment:
nit: no need for the comment

Reply:
Makes sense. I'll get of that . Thanks!

Reply:
Thanks also for the feedback on the PR description. I'm having a few difficulties getting the links to work, so I'll have to check that out when tomorrow.